internal/tiltfile/local_resource.go

```diff
@@ -87,6 +90,11 @@ func (s *tiltfileState) localResource(thread *starlark.Thread, fn *starlark.Buil
 		return nil, fmt.Errorf("local_resource must have a cmd and/or a serve_cmd, but both were empty")
 	}
 
+	links, err := convertLinks(linksVal)
```

Comment:
once this is an Unpacker, this will go away (since starlark will be able to correctly identify which argument failed to parse)